### ISSUE TITLE
Building the Java package with debuild fails at HEAD

### ISSUE THREAD

**Issue by roubert** [googlei18n/libphonenumber#956]
Building the libphonenumber Debian packages directly from a GitHub checkout does no longer work, but fails while building the Java packages:

`$ debuild -i -us -uc -b`

`[ERROR] /usr/local/google/home/roubert/src/libphonenumber/googlei18n/java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java:[302,30] error: package com.google.protobuf.nano does not exist`

(See attached log file for all details.)
[debuild.log.txt](https://github.com/googlei18n/libphonenumber/files/107266/debuild.log.txt)

**Comment by roubert** [comment]
@dpocock I think you were the last one to make changes here. Can it be as easy as a new dependency that needs to be listed somewhere?

**Comment by keghani** [comment]
@roubert 
We moved to nano/Phonemetadata.java and added dependencies (almost everywhere apparently) at release 7.2.1 - likely we missed this.

**Comment by keghani** [comment]
In logs: found protobuf, version 2.5.0

2.5.0 doesn't have nanoproto - to avoid your having to install the required versions we did this hermetically with Maven: https://github.com/googlei18n/libphonenumber/commit/d8268984e4333a6a9215310b9c0c2f376b4cad85

Perhaps something similar can be done with the Debian build.

**Comment by TimurFayruzov** [comment]
I had the same issue when building libphonenumber for java locally and it turns out I had to install the protobuf library in local maven repository, which resolved the problem:
```
mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file -Dfile=../lib/com/google/protobuf/protobuf-java/2.6.1/protobuf-java-2.6.1.jar -DpomFile=../lib/com/google/protobuf/protobuf-java/2.6.1/protobuf-java-2.6.1.pom
```

**Comment by keghani** [comment]
Fixed in 7.6.0 I think?
It would be great if someone could verify :)

See release notes, you may need to add a dependency (but this time it will be standard): https://github.com/googlei18n/libphonenumber/releases/tag/libphonenumber-7.6.0.
